### PR TITLE
Live Simplification

### PR DESCRIPTION
**Pull request by orta**
So, I want Live to be more presentable while we work on it, so I've moved off the fancy lots UI with previews into a separate file, and opened up the space to start working with a simpler model. Currently images are now attached to the pagingVC like normal designs. 

I can come back for the nice animation. This hooks the LiveVC up to work correctly inside a ARSerifNavigationController, and makes it show the toolbars in the same fashion as the auctionVC/auctionInfoVC

Also adds keyboard shortcuts, my mouse is being really unreliable, so I've decided to just not really use it. Left/Right now let you go between

### CODE REVIEW

File: Artsy/App/ARAppStatus.m [hunk 1]
```diff
@@ -50,4 +51,14 @@ + (BOOL)isRunningTests
...
+
++ (BOOL)isOSNineOrGreater
+{
+    static BOOL isNine = NO;
+    static dispatch_once_t onceToken = 0;
+    dispatch_once(&onceToken, ^{
+        isNine = (&UIApplicationOpenURLOptionsAnnotationKey != NULL);
+    });
+    return isNine;
+}
```
Comment (alloy) [inline Artsy/App/ARAppStatus.m]: FYI I don’t think these simple checks really need the overhead of a dispatch.

Comment (orta) [inline Artsy/App/ARAppStatus.m]: Agreed, migrated this and `isRunningTests` to just do their logic

**Comment by orta** [comment]
Alright, I've cleaned it up, and am going for a run :runner:

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by ashfurrow** [comment]
Code looks great to me, I've just cleaned up the merges from master and replaced a few snapshots. Danger failure is "no changelog", which I'm fine merging without for this PR.

**Pull request merged**